static/app/components/events/interfaces/spans/traceErrorList.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
btw you don't need to explicitly import React here since it's a global https://github.com/getsentry/sentry/blob/master/static/app/bootstrap/exportGlobals.tsx#L15

Reply:
Unfortch I need it because I'm using `React.Fragment` below 😢 I guess we're not allowing `<></>` syntax yet?

Reply:
`import {Fragment} from React' would work here

Reply:
yeah we don't allow it. I don't know why though

Reply:
👌🏻 done